desktop_qt_ui/editor/graphics_items.py

```
@@ -712,35 +771,173 @@ def _build_guide_line(self, scene: QGraphicsScene, visible_rect: QRectF, extent:
+            if not isinstance(item, RegionTextItem):
+                continue
+            if item is self or item.isSelected():
+                continue
+            pts = item._get_white_frame_world_points()
+            if pts:
+                others.append(pts)
+        if len(others) < 2:
+            return 0.0, 0.0, []
+
+        threshold = self._spacing_snap_threshold
+        snap_dx = 0.0; snap_dy = 0.0
```

Ruff (0.15.13)

[error] Multiple statements on one line (semicolon)

(E702)